tests/wasmfs/wasmfs_dup.c

```diff
@@ -56,13 +57,18 @@ int main() {
   errno = 0;
   f5 = dup2(f4, -1);
   assert(f5 == -1);
+// These errors are not reported in the JS file system.
+#ifdef WASMFS
   assert(errno == EBADF);
+#endif
```

Comment:
Wait, if this test only runs under wasmfs why do we need these ifdefs or comments?

Either we run the test under both backends (with ifdef) or we run it only under one backend (without ifdefs).    I don't really mind which way you go but we should pick one.

Reply:
Yeah I intentionally left those in as markers in case we actually do change the original file system's behaviour. I feel that the comments are necessary as reminders at least.